docs/spring-boot-runtime/master.adoc

```diff
@@ -133,7 +143,15 @@ include::topics/crud-mission-resources.adoc[leveloffset=+3]
 === {mission-health-check-spring-boot-guide-name}
 :health-check-spring-boot:
 
-include::topics/health-check-mission-intro.adoc[leveloffset=+3]
+include::topics/health-check-mission-intro-paragraph.adoc[leveloffset=+3]
+
+NOTE: If you are not familiar with the health check terminology, see the xref:health-check-concepts_{context}[] section first.
+
+The purpose of this use case is to demonstrate the health check pattern through the use of probing. Probing is used to report the liveness and readiness of an application. In this use case, you configure an application which exposes an HTTP `health` endpoint to issue HTTP requests. If the container is alive, according to the liveness probe on the `health` HTTP endpoint, the management platform receives `200` as return code and no further action is required. If the `health` HTTP endpoint does not return a response, for example if the thread is blocked, then the application is not considered alive according to the liveness probe. In that case, the platform kills the pod corresponding to that application and recreates a new pod to restart the application.
```

Comment:
Similar to comments to above.  Have you considered putting this content in a module?

Reply:
I have, this will be the introduction text in an assembly. (I'll do that in a follow-up PR, I wanted to make the PRs smaller so that they're in more quickly)

Reply:
thank you @tradej